subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/DependencyHandler.java

```diff
@@ -383,6 +383,13 @@
      */
     Dependency gradleApi();
 
+    /**
+     * Creates a dependency on the API of the current version of Gradle.
+     *
+     * @return The dependency.
```

Comment:
It would be good to add some Javadoc on the expected format of the parameter. Are snapshots accepted?

I think we need to mention that only older versions are accepted, right? What happens if I give it a newer or non-existent version?

Reply:
Yeah, probably. Though I wouldn't go into polishing the Javadoc for this spike.